dlrover/trainer/examples/estimator_executor/elasticjobtf2.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: elastic.iml.github.io/v1alpha1
+kind: ElasticJob
+metadata:
+  name: elasticjob-sample-23
+spec:
+  distributionStrategy: ParameterServerStrategy
+  resourceLimits:
+    cpu: "1"
+    memory: "1000Mi"
+  replicaSpecs:
+    ps:
+      replicas: 1
+      restartCount: 3
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: main
+              image: easydl/tf-estimator:iris_dnn_v0
+              imagePullPolicy: IfNotPresent
+              command:
+                - /bin/bash
+                - -c
+                - "pip uninstall dlrover -y && pip install -i https://test.pypi.org/simple/ dlrover==0.1.0rc0.dev6 -U && pip install pyhocon  && cd /usr/local/lib/python3.8/site-packages/dlrover/trainer/examples/estimator_executor  && python -m dlrover.trainer.entry.local_entry --platform=Kubernetes  --conf=conf.TrainConf --enable_easydl=True --ps_num=1 --worker_num=2 --evaluator_num=1" # yamllint disable-line rule:line-length
```

Comment:
we can use  `pip install -i https://test.pypi.org/simple/ dlrover==0.1.0rc0.dev6 -I` to replace 
`pip uninstall dlrover -y && pip install -i https://test.pypi.org/simple/ dlrover==0.1.0rc0.dev6 -U`

Reply:
Can we not set `--ps_num=1 --worker_num=2 --evaluator_num=1` if using auto-scaling?

Reply:
if using auto-scaling, `--ps_num=1 --worker_num=2 --evaluator_num=1` can be removed.